.github/workflows/benchmark.yaml

```
@@ -0,0 +1,85 @@
+name: Benchmark
+
+on:
+  push:
+    branches:
+      - devel
+  pull_request:
+    branches:
+      - devel
+
+jobs:
+  snapshotter:
+    name: "Snapshotter"
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+
+      - name: Install nerdctl
+        run: |
+          VERSION="1.7.7" 
+          wget https://github.com/containerd/nerdctl/releases/download/v$VERSION/nerdctl-full-$VERSION-linux-amd64.tar.gz
+          sudo tar Cxzvvf /usr/local nerdctl-full-$VERSION-linux-amd64.tar.gz
+          sudo systemctl enable --now containerd
+
+      - name: Install cvmfs
+        run: |
+          sudo apt install lsb-release -y
+          if [ ! -f /tmp/cvmfs-release-latest_all.deb ]; then
+              wget https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest_all.deb -O /tmp/cvmfs-release-latest_all.deb
```

```suggestion
              wget https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest_all.deb -O /tmp/cvmfs-release-latest_all.deb
```

We'll want to run this for every commit on master, so I think you have to build cvmfs from source (or run the build as a separate action and move the packages as artifacts)